lisa/sut_orchestrator/azure/common.py

```diff
@@ -442,6 +458,71 @@ def wait_copy_blob(
     log.debug("vhd copied")
 
 
+def connection_string(account: str, key: str):
+    return (
+        "DefaultEndpointsProtocol=https;AccountName="
+        + account
+        + ";AccountKey="
+        + str(key)
+        + ";EndpointSuffix=core.windows.net"
+    )
+
+
+def get_share_service_client(
+    credential: Any,
+    subscription_id: str,
+    account_name: str,
+    resource_group_name: str,
+) -> ShareServiceClient:
+    shared_key_credential = get_storage_credential(
+        credential=credential,
+        subscription_id=subscription_id,
+        account_name=account_name,
+        resource_group_name=resource_group_name,
+    )
+    return ShareServiceClient.from_connection_string(
```

Comment:
As the example, the shared_key_credential is enough to create the ShareServiceClient. It's like to create BlobServiceClient in code.
https://docs.microsoft.com/en-us/azure/developer/python/sdk/storage/azure-storage-file-share/azure.storage.fileshare.shareserviceclient?view=storage-py-v12